src/routes/webhooks/paddle.ts

```diff
@@ -418,6 +418,111 @@ const notifyNewPaddleTransaction = async ({
   await webhooks.transactions.send({ blocks });
 };
 
+const notifyNewPaddleCoresTransaction = async ({
```

Comment:
Can't wait for the first one to fire!! 🤩

Reply:
🚀 soon!